packages/compiler-cli/test/compliance/r3_view_compiler_i18n_spec.ts

```diff
@@ -942,7 +942,7 @@ describe('i18n support in the template compiler', () => {
       verify(input, output);
     });
 
-    it('should sanitize ids and generate proper var names', () => {
+    it('should sanitize ids and generate proper let names', () => {
```

Comment:
https://twitter.com/BenLesh/status/1305699317794181122

```suggestion
    it('should sanitize ids and generate proper var names', () => {
```

Reply:
Hmm I think actually:

```suggestion
    it('should sanitize ids and generate proper variable names', () => {
```

Reply:
Works for me :-)

Reply:
To be fair, I did a find and replace on `"var "`, so `varying` would not have been changed 😛